include/mimick.h

```diff
@@ -26,19 +26,24 @@
 
 # include <stddef.h>
 # include "mimick/preprocess.h"
+# include "mimick/literal.h"
+
+# ifdef __cplusplus
+extern "C" {
+# endif
 
 struct mmk_va_param {
     size_t size;
-    char data[];
+    char data[1];
 };
 
 typedef void(*mmk_fn)(void);
-typedef struct mmk_stub *mmk_stub;
+struct mmk_stub;
```

Comment:
This breaks the API as far as I'm concerned. I'd rather have mmk_stub be an alias for `struct mmk_stub *`. I would be fine if `struct mmk_stub` was renamed to `struct mmk_stub_` however.

Reply:
It does break API, but I could only find one place where the `mmk_stub` alias was being used instead of `struct mmk_stub *`. I figured I'd normalize to the `struct <name>` syntax instead.

Reply:
This is mostly about API usage -- mmk_stub is an opaque type, so having the user type `struct mmk_stub` in C is just misleading because you'd expect to be able to change or read some fields in there.

In other words, I am fine with keeping struct and union keywords when it's possible to access the underlying type, and this is not one of these situations.